src/components/sound.js

```diff
@@ -64,17 +64,29 @@ module.exports.Component = registerComponent('sound', {
     }
     // All sound values set. Load in `src`.
     if (srcChanged) {
-      var self = this;
+      var isStream = (data.src.srcObject instanceof MediaStream);
+
+      if (isStream) {
+        var stream = data.src.srcObject;
+        this.pool.children.forEach(function (sound) {
+          sound.setNodeSource(sound.context.createMediaStreamSource(stream));
+        });
+        this.loaded = true;
+        this.el.emit('sound-loaded');
```

Comment:
do we want to emit which src was emitted? or should developers do the look ups in the event listener and assume nothing's changed since the event was emitted?